tests/generate_test_data/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:latest
+RUN apt-get update
+RUN apt-get -y install curl git libc++-dev jq
+WORKDIR /script
+RUN mkdir artifacts
+COPY ./entrypoint.sh .
+RUN chmod a+x ./entrypoint.sh
+ADD hello_world/ ./hello_world
+RUN chmod -R a+w /script
+ENTRYPOINT [ "/script/entrypoint.sh" ]
```

Comment:
Can you add a line comment to describe what this docker will do? Also I'm suggesting you to add some spaces to split sections:

1. Define the base
2. Install system dependency
3. definitions (in this case just workdir)
4. copy data and set permission
5. define entrypoint